payone-adapter/src/main/java/com/commercetools/payment/payone/config/PayoneConfigurationNames.java

```diff
@@ -20,4 +20,16 @@
     static final String PAID_TO_IBAN = "paidToIBAN";
     static final String PAID_TO_BIC = "paidToBIC";
     static final String PAID_TO_NAME = "paidToAccountHolderName";
+
+    // KLARNA specific options
+    static final String GENDER = "gender"; // KlarnaConstants.Gender
+    static final String IP = "ip"; // IP address, IPv4 or IPv6
+
+    /**
+     * Date of birth, mandatory for Germany, Netherlands and Austria.
+     * The value must be in {@link java.time.LocalDate} parseable format, like <i>YYYY-MM-DD</i>
```

Comment:
In CTP platform it is a Date (JSON date), for Payone request it is a String in own format.

Updated the documentation in previous fix: 5c41b78